lib/mvi/configuration.rb

```diff
@@ -5,20 +5,13 @@
 
 module MVI
   class Configuration < Common::Client::Configuration::SOAP
-    # :nocov:
-    def self.default_mvi_open_timeout
-      Rails.logger.warn 'Settings.mvi.open_timeout not set, using default'
-      2
+    def self.open_timeout
+      Settings.mvi.open_timeout
     end
 
-    def self.default_mvi_timeout
-      Rails.logger.warn 'Settings.mvi.timeout not set, using default'
-      10
```

Comment:
I don't mind removing default like these, provided you can link to the devops tickets here in the PR so we can verify they are properly set. If they are not set for each environment there is a risk of blowing up one of those environments.

Reply:
They're not set in `ansible/deployment/config/vets-api/prod-settings.local.yml.j2` which becomes `config/settings.local.yml`.  We're good because they're set in `config/settings.yml` in this repo, which doesn't get overridden.